src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
-// write code here
+const fields = document.querySelectorAll('.field');
+const labelTemplate = document.createElement('label');
+
+labelTemplate.className = 'field-label';
+
+for (const field of fields) {
+  const lable = labelTemplate.cloneNode();
+  const input = field.lastElementChild;
+  const inputName = formatInputName(input.getAttribute('name'));
+
+  lable.textContent = inputName;
+  lable.htmlFor = input.id;
+
+  input.setAttribute('placeholder', capitalizInputName(inputName));
```

Comment:
```suggestion
  input.placeholder = capitalizInputName(inputName));
```
Так само, ти можеш просто присвоїти у властивість placeholder нове значення. Це працюватиме так само як setAttribute